src/storage/noteStorage.js

```diff
@@ -92,6 +93,19 @@ function remove(note) {
     });
 }
 
+function move({ data }) {
```

Comment:
Which props do we expect in `data`?

If someone would need to use this method, how should he figure out what to pass?